spec/features/beis_users_can_download_exports_spec.rb

```diff
@@ -169,6 +171,7 @@
         "Partner organisation" => partner_organisation.name,
         "Level" => "Project (level C)",
         "Title" => project.title,
+        "Level B activity comments" => "",
```

Comment:
Might be worth adding a single comment here, just so it's covered for both types of download.

Reply:
Left this in as a fixup for now - I think we should test this, but it does mean adding a programme, budget and comment